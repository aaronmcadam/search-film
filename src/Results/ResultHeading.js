import styled from 'styled-components';

const ResultHeading = styled.h2`
  font-size: 2rem;
  margin: 0;
  margin-bottom: 1rem;
`;

export default ResultHeading;
