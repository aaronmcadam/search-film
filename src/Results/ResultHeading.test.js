import React from 'react';
import { shallow } from 'enzyme';
import ResultHeading from './ResultHeading';

const setup = () => {
  const wrapper = shallow(<ResultHeading />);

  return {
    wrapper,
  };
};

test('renders', () => {
  const { wrapper } = setup();

  expect(wrapper).toMatchSnapshot();
});
