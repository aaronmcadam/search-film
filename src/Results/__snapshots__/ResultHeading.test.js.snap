// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <h2
    className="sc-bdVaJa bdJBWx"
/>,
  "nodes": Array [
    <h2
      className="sc-bdVaJa bdJBWx"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <styled.h2 />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StyledComponent {
        "_reactInternalInstance": [Circular],
        "attrs": Object {},
        "context": Object {
          "__styled-components-stylesheet__": undefined,
          "__styled-components__": undefined,
        },
        "props": Object {},
        "refs": Object {},
        "state": Object {
          "generatedClassName": "bdJBWx",
          "theme": Object {},
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <h2
          className="sc-bdVaJa bdJBWx"
/>,
        "_debugID": 2,
        "_renderedOutput": <h2
          className="sc-bdVaJa bdJBWx"
/>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <styled.h2 />,
}
`;
